sdk/monitor/opentelemetry-exporter-azure-monitor/src/platform/nodejs/persist/noopPersist.ts

```diff
@@ -0,0 +1,10 @@
+import { PersistentStorage } from "../../../types";
+
+export class NoopPersist implements PersistentStorage {
```

Comment:
where does this get used?

Reply:
I'll remove this since it is an internal "demo" class only